distribution/run.yaml

```diff
@@ -122,18 +122,18 @@ providers:
     provider_type: remote::trustyai_lmeval
     module: llama_stack_provider_lmeval==0.2.4
     config:
-        use_k8s: ${env.TRUSTYAI_LMEVAL_USE_K8S:=true}
-        base_url: ${env.VLLM_URL:=http://localhost:8000/v1}
-  - provider_id: ${env.EMBEDDING_MODEL:+trustyai_ragas_inline}
+      use_k8s: ${env.TRUSTYAI_LMEVAL_USE_K8S:=true}
+      base_url: ${env.VLLM_URL:=http://localhost:8000/v1}
+  - provider_id: ${env.TRUSTYAI_EMBEDDING_MODEL:+trustyai_ragas_inline}
     provider_type: inline::trustyai_ragas
     module: llama_stack_provider_ragas.inline
     config:
-      embedding_model: ${env.EMBEDDING_MODEL:=}
+      embedding_model: ${env.TRUSTYAI_EMBEDDING_MODEL:=}
   - provider_id: ${env.KUBEFLOW_LLAMA_STACK_URL:+trustyai_ragas_remote}
     provider_type: remote::trustyai_ragas
     module: llama_stack_provider_ragas.remote
     config:
-      embedding_model: ${env.EMBEDDING_MODEL:=}
+      embedding_model: ${env.TRUSTYAI_EMBEDDING_MODEL:=}
```

Comment:
how is this model going to be served? i don't see it being optionally registered down below, is it expected?